etl/steps/data/garden/bp/2022-07-11/statistical_review.meta.yml

```diff
@@ -3,7 +3,6 @@ dataset:
   version: 2022-07-11
   title: Statistical Review of World Energy - BP (2021)
   short_name: statistical_review
-  publication_year: 2021
   sources:
     -
       name: Our World in Data based on BP Statistical Review of World Energy (2021)
```

Comment:
Would it be possible to impose that dates must always be in the format "YYYY-MM-DD", to avoid things like "8th July 2021" here?